prow/config/secrets_agent.go

```diff
@@ -89,7 +90,7 @@ func (sa *SecretAgent) GetSecret(secretPath string) []byte {
 	return sa.secretsMap[secretPath]
 }
 
-// Set sets the map of secrets.
+// SetSecret sets the map of secrets.
 func (sa *SecretAgent) SetSecret(secretPath string, secretValue []byte) {
```

Comment:
Is this function unused now?

Reply:
stop using it after some refactoring. I will remove it.

Reply:
If `SetSecret` is only used by `reloadSecret` lets avoid exporting it.